src/pages/forms/[id].tsx

```diff
@@ -61,6 +61,11 @@ export default function FormDetailPage() {
     : 'bg-red-100 text-red-800';
   const camelCaseTitle =
     form.name.charAt(0).toUpperCase() + form.name.slice(1, form.name.length);
+
+  const closeForm = async () => {
```

Comment:
Can you implement this function using the logic from `src/utils.ts` and then delete the function in `src/utils.ts`?